src/lib.rs

```diff
@@ -2092,6 +2092,16 @@ impl Build {
                         cmd.push_cc_arg("-pthread".into());
                     }
                 }
+
+                if target.os == "nto" {
+                    let arg = match target.arch {
+                        "i586" => "-Vgcc_ntox86_cxx",
+                        "aarch64" => "-Vgcc_ntoaarch64le_cxx",
+                        "x86_64" => "-Vgcc_ntox86_64_cxx",
+                        _ => panic!("Unknown architecture for Neutrino QNX: {}", target.arch),
```

Comment:
```suggestion
                        _ => return Err(Error::new(ErrorKind::InvalidTarget, format!("Unknown architecture for Neutrino QNX: {}", target.arch)),
```

Let's not use panic, the code snippet would need cargo-fmt though

Reply:
Good suggestion, thanks. Done.